lg69t/firmware_tool.py

```diff
@@ -163,7 +165,7 @@ class UpgradeType(Enum):
     GNSS = auto()
 
 
-def Upgrade(port_name: str, bin_path: str, upgrade_type: UpgradeType, should_send_reboot: bool):
+def Upgrade(port_name: str, bin_file, upgrade_type: UpgradeType, should_send_reboot: bool):
```

Comment:
You've done away with the type hint here. You ought to keep that.

Reply:
I believe `typing.BinaryIO` is the proper type.